core/src/commonMain/kotlin/com/islamversity/core/Logger.kt

```diff
@@ -0,0 +1,63 @@
+package com.islamversity.core
+
+import co.touchlab.kermit.CommonLogger
+import co.touchlab.kermit.Kermit
+import co.touchlab.kermit.Logger
+import com.islamversity.core.Logger.init
+import co.touchlab.kermit.Severity as KSeverity
+
+enum class Severity {
```

Comment:
😰 enum

Reply:
why not?

Reply:
you mean I should create functions instead?

Reply:
You know, a single enum is 13x bigger than the Integer constant